### DISCUSSION TITLE
How can I use postgres function?

### DISCUSSION THREAD

**Discussion by duskmoon314** [rbatis/rbatis#290] [General]
I would like to use uuid type in postgres and I want to use `gen_random_uuid` provided by postgres. How can I make it? Is there any example?

It seems like that I need to use html_sql or pysql for now, right?

**Comment by zhuxiujia** [comment]
> I would like to use uuid type in postgres and I want to use `gen_random_uuid` provided by postgres. How can I make it? Is there any example?
> 
> It seems like that I need to use html_sql or pysql for now, right?

yes.use html_sql or pysql.

Another option is to use the built-in Format feature.see https://rbatis.github.io/rbatis.io/#/en/?id=crudtable
